### PR TITLE
docs(aws-machine-image): add admonition

### PR DESCRIPTION
**Pull request by HonkingGoose**
## Changes

- Add `warning` admonition

## Context

This draws more attention than just using **bold** and _italics_.
The text is easier to read, because it's just regular text now. 😄 

Helps with #13665.

## Documentation (please check one with an [x])

- [x] I have updated the documentation, or
- [ ] No documentation update is required

## How I've tested my work (please tick one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: lib/modules/datasource/aws-machine-image/readme.md [hunk 1]
```diff
@@ -1,4 +1,7 @@
-> :warning: **This datasource is _experimental_**: Be aware that its syntax and behavior may change at any time!
+<!-- prettier-ignore -->
```
Comment (HonkingGoose) [inline lib/modules/datasource/aws-machine-image/readme.md:1]: Do we publish this readme on our docs site? I cannot find it when doing a search on the production site.

Comment (HonkingGoose) [reply lib/modules/datasource/aws-machine-image/readme.md:1]: Do we copy other readmes into the published docs? Or is it just this one that we forgot in a build-script somewhere?

Comment (viceice) [reply lib/modules/datasource/aws-machine-image/readme.md:1]: We don't read those files yet. We should do this for datasources / versionings and platforms, like we do for managers.

Comment (HonkingGoose) [reply lib/modules/datasource/aws-machine-image/readme.md:1]: I found issue #8855, should we add the "copy/paste `readme.md` into published docs" step in there? Or do you want a completely new issue to track the idea?

Comment (rarkins) [reply lib/modules/datasource/aws-machine-image/readme.md:1]: I think new issue, it can be solved independently

Comment (HonkingGoose) [reply lib/modules/datasource/aws-machine-image/readme.md:1]: I made issue #14608 to track the work needed to publish the `readme.md` files. 😉

**Comment by HonkingGoose** [comment]
This is `status:blocked` until we publish this file in our docs, see issue #14608.

**Comment by viceice** [comment]
Anything else blocking this?

**Comment by HonkingGoose** [comment]
We now publish the `readme.md`: https://docs.renovatebot.com/modules/datasource/#aws-machine-image-datasource

I think we're good to go after review. 😄

**Pull request merged**